CMakeLists.txt.user

```diff
@@ -0,0 +1,319 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
This file should not be part of a PR it contains local settings that are not relevant.

Reply:
ahh sorry didnt check git diff first.